clientv3/maintenance.go

```diff
@@ -50,6 +51,11 @@ type Maintenance interface {
 	// Status gets the status of the endpoint.
 	Status(ctx context.Context, endpoint string) (*StatusResponse, error)
 
+	// HashKV returns the state of backend database at the time of the RPC.
+	// If revision is non-zero, hash is computed only in key bucket and keys at or below
```

Comment:
I wonder should the comment for maintenance HashKV the same as the one in rpc?
`HashKV computes the hash of all MVCC keys up to a given revision.`
https://github.com/coreos/etcd/blob/master/etcdserver/etcdserverpb/rpc.proto#L187